### ISSUE TITLE
[BUG]  ask for one-click_nuclei but modelt set to denoise_cyto3

### ISSUE THREAD

**Issue by pamaiuri** [MouseLand/cellpose#1091]
**Describe the bug**
Both on the CellPose gui or from command line, when I ask for image restoration 'one-click' nuclei ( --restore_type 'oneclick_nuclei' on the command line) I got no error but it seams to load the default metod.

This is the log during the operation on the gui:
**one-click_nuclei**
**2025-01-22 16:41:09,502 [INFO] >> denoise_cyto3 << model set to be used**
2025-01-22 16:41:09,503 [INFO] >>>> using CPU
2025-01-22 16:41:09,503 [INFO] >>>> using CPU
2025-01-22 16:41:09,706 [INFO] >>>> model diam_mean =  30.000 (ROIs rescaled to this size during training)
GUI_INFO: channels:  [0, 0]
GUI_INFO: normalize_params:  {'lowhigh': None, 'percentile': [1.0, 99.0], 'normalize': True, 'norm3D': True, 'sharpen_radius': 0, 'smooth_radius': 0, 'tile_norm_blocksize': 0, 'tile_norm_smooth3D': 1, 'invert': False}
GUI_INFO: diameter (before upsampling):  30.0
(1, 2048, 2048, 1)
(2048, 2048, 1)
2025-01-22 16:41:19,777 [INFO] **one-click_nuclei** finished in 10.278 sec

This the one from command line (--restore_type 'oneclick_nuclei'):
2025-01-22 15:53:07,815 [INFO] >>>> running cellpose on 1 images using chan_to_seg GRAY and chan (opt) NONE
2025-01-22 15:53:07,815 [INFO] >> **denoise_cyto3** << model set to be used
/home/PM_Application/CellPose/cellpose/lib/python3.12/site-packages/cellpose/resnet_torch.py:275: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  state_dict = torch.load(filename, map_location=torch.device("cpu"))
2025-01-22 15:53:08,019 [INFO] >>>> model diam_mean =  30.000 (ROIs rescaled to this size during training)
2025-01-22 15:53:08,019 [INFO] >> nuclei << model set to be used
2025-01-22 15:53:08,060 [INFO] >>>> loading model /home/paolo/.cellpose/models/nucleitorch_0
2025-01-22 15:53:08,128 [INFO] >>>> model diam_mean =  17.000 (ROIs rescaled to this size during training)
2025-01-22 15:53:08,128 [INFO] >>>> using diameter 20.000 for all images

Is it just wrong message or for some reason I did a mistake and I didn't manage to set the desired model?

Thank you for your help.

**Comment by pamaiuri** [comment]
update, from command line (--restore_type 'oneclick_nuclei') we got now this:

2025-01-24 15:33:05,812 [INFO] Downloading: "https://www.cellpose.org/models/oneclick_nuclei" to /home/maky/.cellpose/models/**oneclick_nuclei**
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 25.4M/25.4M [00:03<00:00, 8.32MB/s]
2025-01-24 15:33:09,575 [INFO] >>>> running cellpose on 1 images using chan_to_seg GRAY and chan (opt) NONE
2025-01-24 15:33:09,576 [INFO] >> **denoise_cyto3** << model set to be used

thank you all for help.

**Comment by carsen-stringer** [comment]
sorry this is fixed in the latest github version, please upgrade manually (pip install git+https://github.com/mouseland/cellpose.git) or wait a few days for the new pip

**Comment by pamaiuri** [comment]
thank you a lot!
